cosmic-applet-audio/src/main.rs

```diff
@@ -313,12 +320,13 @@ impl Application for Audio {
                     .padding([8, 24]),
                     row![
                         icon("audio-input-microphone-symbolic", 32)
-                            .width(Length::Units(24))
-                            .height(Length::Units(24))
+                            .width(Length::Fixed(24.0))
```

Comment:
why does the `icon` helper need a size in the function call, then a resize in iced?

Reply:
Honestly, I don't remember. I'll check and add a comment if it is indeed necessary.